gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-kotlin = "1.8.10"
+kotlin = "1.9.20-Beta"
```

Comment:
Is it part of fix or is it just following the last version?

Reply:
just updating.